### DISCUSSION TITLE
How to make popup not affecting other elements (not overlaying)

### DISCUSSION THREAD

**Discussion by DilemmaGX** [microsoft/fluentui#31650] [Q&A] [answered]
```typescript
  <Menu>
    <MenuTrigger disableButtonEnhancement>
      <MenuButton>Example</MenuButton>
    </MenuTrigger>

    <MenuPopover>
      <MenuList>
        <MenuItem>Item a</MenuItem>
        <MenuItem>Item b</MenuItem>
      </MenuList>
    </MenuPopover>
  </Menu>
```
I put these in my App.tsx file, however, after triggering the menu, there's a white overlay that covers everything else (including the trigger) up. I tried making it into another tsx file and import it, but the overlaying still occures.
Last time I tried the storybook's dialog example on my file, it also generats a white overlay.
How to fix this?

**Comment by ZakariaWijaya** [comment]
exactly the same problem I had, I'm using fluentui v9 on my nextjs project. i'm trying in stackblitz.com is worked

![image](https://github.com/microsoft/fluentui/assets/109570246/aff1cece-b418-45e2-93fd-f9caa9096efb)

**Comment by ZakariaWijaya** [comment]
I tried inspect element, something happened to the expanded property but the popup didn't appear.

![image](https://github.com/microsoft/fluentui/assets/109570246/2642b33e-ac9a-4c6d-9c6f-04176534952e)

**Comment by DilemmaGX** [comment, marked as answer]
Well I think I know how now...
When using `FluentProvider`, add attribute `applyStylesToPortals={false}`.
That is:
```html
<FluentProvider theme={webLightTheme} applyStylesToPortals={false}></FluentProvider>
```
I'm not sure about the side effects, though.

**Comment by AdamBlack34** [comment]
This exact scenario happened to me this morning. Across my app I have various places where I am using popovers but when the popover appears it takes over the whole screen. Using applyStylesToPortals={false} fixes this but I do not understand the root cause, has anyone figured this out?
